snippets/src/language/normalize_comments.rs

```diff
@@ -0,0 +1,66 @@
+use crate::tree_sitter_consts::NODE_KIND_COMMENT;
+use std::borrow::Cow;
+
+use super::snippet_context::SnippetContext;
+
+/// Remove all comment node text from the given content.
+/// In general, this function should work in any language that produces "comment" type nodes from tree_sitter.
+#[tracing::instrument(skip_all)]
```

Comment:
[nit] Prefer to name the literal. e.g.:
```suggestion
/// In general, this function should work in any language for which treesitter produces nodes whose `type` fields equal [`NODE_KIND_COMMENT`].
#[tracing::instrument(skip_all)]
```